apps/web/components/integrations/IntegrationListItem.tsx

```diff
@@ -15,7 +14,7 @@ function IntegrationListItem(props: {
   return (
     <ListItem expanded={!!props.children} className={classNames("flex-col")}>
       <div className={classNames("flex w-full flex-1 items-center space-x-2 p-3 rtl:space-x-reverse")}>
-        <Image width={40} height={40} src={`/${props.imageSrc}`} alt={props.title} />
+        {props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
```

Comment:
We are handling remote url logos form apps, to achieve this with Next/Image we need to add available domains in our next.config.js images: []
https://nextjs.org/docs/messages/next-image-unconfigured-host